prjxray/fasm_assembler.py

```diff
@@ -0,0 +1,152 @@
+import fasm
```

Comment:
wasn't fasm stuff getting moved to an external library that called prjxray lib functions?

Reply:
I had planned it the other way.  The FASM library handles parsing and emission of FASM itself.  Each target handles how to take parsed FASM and do something with it.  I see a lot more variation in each target.  I guess we could move to a plugin model, but that is tricky to design, I don't think I'd want to try to get that right on the first go.

Reply:
The library is pretty boring.  It's pretty much string -> AST, AST -> string, and partial canonization.